Samples/ReVerC/Operation.qs

```diff
@@ -0,0 +1,44 @@
+﻿namespace Quantum.ReVerC
+{
+    open Microsoft.Quantum.Primitive;
+    open Microsoft.Quantum.Canon;
+
+  /// Prepares a quantum register in the given basis state
+  operation prepare(initial: Result[], register: Qubit[]) : ()
+  {
+    body
+    {
+      for (i in 0..Length(initial)-1)
```

Comment:
Generally, we try to avoid single-letter variable names. I'd suggest using `idxQubit` here to emphasize what is being indexed.